tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java

```diff
@@ -159,13 +159,13 @@ public void testSimpleWebAppWithJSP() throws Exception
         };
         try (JettyHomeTester.Run run1 = distribution.start(args1))
         {
-            assertTrue(run1.awaitFor(5, TimeUnit.SECONDS));
+            assertTrue(run1.awaitFor(10, TimeUnit.SECONDS));
             assertEquals(0, run1.getExitValue());
 
             // Verify that --create-start-ini works
             assertTrue(Files.exists(jettyBase.resolve("start.ini")));
 
-            File war = distribution.resolveArtifact("org.eclipse.jetty.tests:test-simple-webapp:war:" + jettyVersion);
+            File war = distribution.resolveArtifact("org.eclipse.jetty.demos:demo-simple-jsp-webapp:war:" + jettyVersion);
```

Comment:
Same as previous comment - think this should be either demo-jsp-webapp or demo-simple-webapp, not sure which.